### ISSUE TITLE
equamax rotation

### ISSUE THREAD

**Issue by maxheld83** [aiorazabala/qmethod#28]
> Equamax is an orthogonal rotation method that combines varimax and quartimax techniques. It simultaneously minimizes both the number of variables and the number of factors.
> (Source: [Noori Akhtar-Danesh](http://fhs.mcmaster.ca/ceb/faculty_member_akhtar-danesh.htm) presentation at q conference in 2007)

Currently, **psych** `principal` only includes:
> "none", "varimax", "quatimax", "promax", "oblimin", "simplimax", and "clus- ter" are possible rotations/transformations of the solution.

**Comment by aiorazabala** [comment]
UPDATE: The package `psych` supports `equamax` rotation since version 1.4.6 (http://r.789695.n4.nabble.com/Principal-component-analysis-with-EQUAMAX-rotation-td4692337.html#a4693612 by the author of `psych`)

This is not in their documentation page, but you can specify the argument `rotate='equamax'` in their function `principal()` and it doesn't return an error of `unused argument`, hence one assumes that the rotation is run (I believe that the rotation modes are taken from package `GPArotation`.

``{r}
# e.g.
pc <- principal(Harman74.cor$cov, 4, rotate = "equamax")
```

The function `qmethod()` accepts all the values of rotate that `principal()` can accept, thus you can run it with `equamax`. However I can give no guarantee of whether what comes out is sensible.

**Comment by aiorazabala** [comment]
A quick test of rotation `varimax` with rotation `equamax` and dataset `lipset` gives the following comparison:
* Eigenvalues and explained variance slightly more equal across the rotated factors
* z-score differences: maximum 0.1, on average 0.025
* Factor score differences: maximum 1, on average 0.12
* Rotated factor loading differences: maximum 0.11, on average 0.04

**Comment by maxheld83** [comment]
That's great news – so we can have this new feature now, without extra code required. 

Thanks for validating this so quickly!

I'll make sure to test this with my dataset, too, and report the differences here.
They seem to be pretty modest.

> I believe that the rotation modes are taken from package GPArotation.
I think I read that in the `psych` manual, too.

**Comment by maxheld83** [comment]
@aiorazabala will write an e-mail to Noori updating him.
In future, once principal-axis factoring is also implemented #20 we should update him again.
